test/boost/cdc_smp1_test.cc

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2020 ScyllaDB
+ */
+
+/*
+ * This file is part of Scylla.
+ *
+ * Scylla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Scylla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "cdc/log.hh"
+#include "db/config.hh"
+#include "cql3/query_options.hh"
+#include "test/lib/cql_assertions.hh"
+#include "test/lib/cql_test_env.hh"
+#include "transport/messages/result_message.hh"
+
+#include <seastar/testing/thread_test_case.hh>
+
+static std::vector<std::vector<bytes_opt>> to_bytes(const cql_transport::messages::result_message::rows& rows) {
+    auto rs = rows.rs().result_set().rows();
+    std::vector<std::vector<bytes_opt>> results;
+    for (auto it = rs.begin(); it != rs.end(); ++it) {
+        results.push_back(*it);
+    }
+    return results;
+}
+
+static cql_test_config mk_lwt_cdc_test_config() {
+    shared_ptr<db::config> cfg(make_shared<db::config>());
```

Comment:
You will have to register cdc schema extension in order for this test to work after it is rebased onto next. See [this code](https://github.com/scylladb/scylla/blob/d1db1982113e331c9ca092cfa2454ec78b942f33/test/boost/cdc_test.cc#L46) for reference.